src/test/regress/sql/multi_subquery_complex_queries.sql

```diff
@@ -2428,3 +2428,67 @@ FROM (users_table u FULL JOIN events_table e USING (user_id)) k(x1, x2, x3, x4,
 ORDER BY 1, 2, 3
 LIMIT 5;
 
+SELECT c_custkey
+FROM  (users_table LEFT OUTER JOIN events_table ON (users_table.user_id = events_table.user_id)) AS test(c_custkey, c_nationkey)
+	INNER JOIN users_table as u2 ON (test.c_custkey = u2.user_id)
+ORDER BY 1 DESC
+LIMIT 10;
+
+SELECT c_custkey, date_trunc('minute', max(c_nationkey))
+FROM  (users_table LEFT OUTER JOIN events_table ON (users_table.user_id = events_table.user_id)) AS test(c_custkey, c_nationkey)
+	INNER JOIN users_table as u2 ON (test.c_custkey = u2.user_id)
+GROUP BY 1
+ORDER BY 2, 1
+LIMIT 10;
+
+SELECT c_custkey, date_trunc('minute', max(c_nationkey))
```

Comment:
I think you forgot to add the FULL JOIN test that you've pushed the fix for?
```SQL
SELECT user_id
FROM (users_table
      JOIN events_table USING (user_id)) AS test(user_id, c_nationkey)
FULL JOIN users_table AS u2 USING (user_id)
LIMIT 10;
```

Reply:
added